tensorboard/pip_package/requirements.txt

```diff
@@ -20,14 +20,14 @@ absl-py >= 0.4
 grpcio >= 1.48.2
 markdown >= 2.6.8
 numpy >= 1.12.0
+packaging
 # NOTE: this version must be >= the protoc version in our WORKSPACE file.
 # At the same time, any constraints we specify here must allow at least some
 # version to be installed that is also compatible with TensorFlow's constraints:
 # https://github.com/tensorflow/tensorflow/blob/25adc4fccb4b0bb5a933eba1d246380e7b87d7f7/tensorflow/tools/pip_package/setup.py#L101
 # 4.24.0 had an issue that broke our tests, so we should avoid that release:
 # https://github.com/protocolbuffers/protobuf/issues/13485
-# 5.26.0 introduced a breaking change, so we restricted it for now. See issue #6808 for details.
-protobuf >= 3.19.6, != 4.24.0, < 5.0.0
+protobuf >= 3.19.6, != 4.24.0
```

Comment:
Do you think we should add a TODO or a comment somewhere, to remove the `packaging` dependency and restrict this to >= 5.0.0 after it's been available for a while or something?

I worry that if at some point we remove this logic to check for protobuf version, we won't realize that we also could remove the packaging dependency, as it was only used there. Maybe there is some tooling that would let us know this... maybe we can add a small comment right next to that package. WDYT?

Reply:
Good call, added a comment specifying that if we bump protobuf >= 5.0.0, we can also remove the packaging dependency.